src/main/java/com/devcycle/sdk/server/common/api/DVCApiClient.java

```diff
@@ -1,40 +1,76 @@
 package com.devcycle.sdk.server.common.api;
 
 import com.devcycle.sdk.server.common.interceptor.AuthorizationHeaderInterceptor;
+import com.devcycle.sdk.server.common.model.DVCOptions;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
 import okhttp3.*;
 import retrofit2.Retrofit;
 import retrofit2.converter.jackson.JacksonConverterFactory;
 
 import java.util.Objects;
+import java.util.concurrent.TimeUnit;
 
 public final class DVCApiClient {
 
   private final OkHttpClient.Builder okBuilder;
   private final Retrofit.Builder adapterBuilder;
 
-  private static final String BASE_URL = "https://bucketing-api.devcycle.com/";
+  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
 
-  private DVCApiClient() {
-    okBuilder = new OkHttpClient.Builder();
+  private static final String BUCKETING_URL = "https://bucketing-api.devcycle.com/";
+  private static final String CONFIG_URL = "https://config-cdn.devcycle.com/";
+  private static final int DEFAULT_TIMEOUT_MS = 10000;
+  private static final int MIN_INTERVALS_MS = 1000;
```

Comment:
Is this the same minimum everywhere else?

Reply:
yup, these are the defaults... just noticed we have a slight mismatch for the timeout compared to the NodeJS SDK, will update it to match (timeout maximum should be the polling interval, will keep the minimum of 1,000 ms)